SConstruct

```diff
@@ -15,7 +14,7 @@ try:
 except NameError:
     pass
 
-sys.path.insert(0, os.path.join(Dir('#').abspath, 'utils/sl'))
+from prereq_tools import PreReqComponent
```

Comment:
(pylint-import-error) Unable to import 'prereq_tools'

Reply:
check_script.py creates a new SConscript file and adds lines to account of things that are taken for granted.   We could potentially add the line above back in the wrapper to avoid this warning

Reply:
This warning is already fixed via adding site_scons to PYTHONPATH, you can just still see it in github because I haven't updated this line since.